dao/src/main/java/com/epam/esm/dao/repository/UserRepository.java

```diff
@@ -1,16 +1,16 @@
 package com.epam.esm.dao.repository;
 
-import com.epam.esm.dao.entity.User;
+import com.epam.esm.dao.entity.Users;
 
 import java.util.Optional;
 
 /** * This interface describes a common operations with User's Entities situated in the DB */
-public interface UserRepository extends Repository<User> {
+public interface UserRepository extends Repository<Users> {
   /**
    * * Updating the Gift Certificate with the parameters passed in the Entity
    *
-   * @param user - passed into the method User that is contained in one of all tables in the DB
+   * @param users - passed into the method User that is contained in one of all tables in the DB
    * @return Optional<User> - container that is contained User
    */
-  Optional<User> update(User user);
+  Optional<Users> update(Users users);
```

Comment:
Are you updating the list of users? name 'users'

Reply:
Yes, this is my mistake. Thank you. (fixed)